javascript/javascript1/week3/homework3.js

```diff
@@ -0,0 +1,153 @@
+////////////////////////////////////////////////////////////////////////////item array removal
+
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+
+// Write some code here
+
+for (var i = 0; i < names.length; i++) {
+    if (names[i] === nameToRemove) {
+        names.splice(i, 1);
```

Comment:
We should probably break out of the loop after we've removed the items to avoid having to go through all of them. Your solution is perfectly valid and it's the best especially if we have several occurrences of 'Ahmed' or the `nameToRemove` in the collection of names